Arrowgene.Ddon.Shared/Entity/Structure/CDataStageLayoutEnemyPresetEnemyInfoClient.cs

```diff
@@ -49,6 +49,7 @@ public CDataStageLayoutEnemyPresetEnemyInfoClient()
         public bool IsAreaBoss { get; set; }
         public bool IsBloodEnemy { get; set; }
         public bool IsHighOrbEnemy { get; set; }
+        public string SpawnTime { get; set; }
```

Comment:
Instead of storing the string, the converted numeric values should be the ones being stored

Reply:
Nevermind, this is a CData class, you shouldnt add anything new to it